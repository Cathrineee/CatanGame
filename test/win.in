fair
roll
build-road 33
build-road 41
improve 0
improve 0 
build-res 34
build-road 45
build-road 42
build-res 29
improve 22
improve 22
improve 34
improve 34
no
