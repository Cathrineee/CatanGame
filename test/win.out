                    |BH|-- 0--| 1|                    
                      |         |                     
                      1    0    2                     
                      |  GLASS  |                     
          |YB|-- 3--| 3|   2  |RB|-- 4--| 5|          
            |         |         |         |           
            5    1    6         7    2    8           
            |  ENERGY |         |  WIFI   |           
| 6|-- 9--| 7|   5  |OB|--10--| 9|   6  |OB|--11--|11| 
  |         |         |         |         |         | 
 12    3   13        14    4   15        16    5   17 
  |  ENERGY |         |  GLASS  |         |  BRICK  | 
|12|   3  |13|--18--|14|   3  |RB|--19--|16|   3  |YB|
  |         |         |         |         |         | 
 20        21    6   22        23    7   24        25 
  |         |  HEAT   |         |  BRICK  |         | 
|18|--26--|19|   3  |20|--27--|21|   4  |BB|--28--|23|
  |         |         |         |         |         | 
 29    8   30        31    9   32        33   10   34 
  |  GLASS  |         |  ENERGY |         |  WIFI   | 
|24|   4  |25|--35--|26|  11  |27|--36--|28|  11  |29|
  |         |         |         |         |         | 
 37        38   11   39        40   12   41        42 
  |         |  BRICK  |         |  BRICK  |         | 
|30|--43--|31|   8  |32|--44--|33|   8  |34|--45--|35|
  |         |         |         |         |         | 
 46   13   47        48   14   49        50   15   51 
  |  HEAT   |         |  HEAT   |         |  ENERGY | 
|36|  11  |37|--52--|38|   8  |39|--53--|40|  11  |41|
  |         |         |         |         |         | 
 54        55   16   56        57   17   58        59 
  |         |  WIFI   |         |  PARK   |         | 
|42|--60--|43|   9  |44|--61--|45|      |46|--62--|47|
            |         |         |         |           
           63        64   18   65        66           
            |         |  GLASS  |         |           
          |48|--67--|49|  12  |50|--68--|51|          
                      |         |                     
                     69        70                     
                      |         |                     
                    |52|--71--|53| 
Builder Blue's turn.
> Builder Blue now has fair Dice.
> The number you rolled is 5
Builder YELLOW gained:
1 ENERGY
Builder ORANGE gained:
1 ENERGY
Enter a command:
> Builder BLUE has successfully built a road at 33.
Enter a command:
> Builder BLUE has successfully built a road at 41.
Enter a command:
> Builder BLUE has successfully built a tower at 0.
Enter a command:
> Enter a command:
> Builder BLUE has successfully built a basement at 34.
Enter a command:
> Builder BLUE has successfully built a road at 45.
Enter a command:
> Builder BLUE has successfully built a road at 42.
Enter a command:
> Builder BLUE has successfully built a basement at 29.
Enter a command:
> Builder BLUE has successfully built a house at 22.
Enter a command:
> Builder BLUE has successfully built a tower at 22.
Enter a command:
> Builder BLUE has successfully built a house at 34.
Enter a command:
> Builder BLUE has successfully built a tower at 34.
Congratulations to Blue! You have won the game!
Builder Blue, would you like to play again?
